src/Assignment-4/assignment4.css

```diff
@@ -5,3 +5,109 @@
 width: 100%;
 height: 100%;
 }
+body {
+    font-family: 'Raleway', sans-serif;
+}
+
+header{
+background: #92E0E5;
+height:50px;
+padding:0px 25px;
+color:white;
+}
+header h1{
+    float: left;
+    text-indent: -9999px;
+    background: url("./assets/images/icon_dog.png") no-repeat;
+    background-size: cover;
+    border-radius:50%;
+    height: 35px;
+    width: 35px;
+    margin-top:8px;
+}
+header nav{
+    float:right;
+}
+nav ul li {
+ display:inline-block;
+ padding:17px 11px;
+}
+nav ul li:hover{
+background: #3F8DA7;
+color:white;
+cursor:pointer;
+}
+a{
+    text-decoration:none;
+}
+#hero{
+    background:url("./assets/images/park.jpg")no-repeat;
+    background-size: cover;
+    text-align: center;
+    padding: 50px 0px;
+    margin-bottom: 50px;
+}
+#hero img {
+    display: block;
+    margin: auto;
+    margin-bottom: 25px;
+    border-radius: 50%;
+    border: 5px solid #3F8DA7;
+}
+#hero h2 {
+    font-size: 30px;
+    margin-bottom:25px;
+}
+
+.button {
+    background: white;
+    border: 5px solid #3F8DA7;
+    color: #3F8DA7;
+    padding: 5px 10px;
+    text-decoration: none;
+    margin-top:50px;
+
+}
+.button:hover {
+    padding: 5px 10px;
+    background: #3F8DA7;
+    color: white;
+} 
+.three-col{
+    display: inline-block;
+    width:30%;
+    vertical-align:top;
+    text-align: center;
+    margin:0px 10px;
+}
+.three-col img {
+    border-radius: 50%;
+    border: 3px solid #3F8DA7;
+    height: 200px;
+    width: 200px;
+    margin: 25px 0px;
+}
+.three-col p{
+    margin-bottom: 40px;
+}
+footer{
+    background:#3F8DA7;
+    color:white;
+    margin:40px;
```

Comment:
just use only margin-top it looks good. no need to use all other margins..for footer